### ISSUE TITLE
non-English system language - syntax error

### ISSUE THREAD

**Issue by denfromufa** [3F/DllExport#7]
https://github.com/pythonnet/pythonnet/issues/220

the issue is:

```
syntax error at token '.field' in: .field static assembly initonly int64
```

and the manual solution is:

> yajiedesign commented 3 days ago
> ok,I found and fixed the problem.
> RGiesecke.DllExport does not support non English ILdasm generated IL file.
> you can pathed RGiesecke.DllExport with ilspy fixed it
> find and change "// Code" to you system language.
> it in RGiesecke.DllExport.Parsing.Actions.DeleteExportAttributeParserAction.Execute

**Comment by 3F** [comment]
Please use pull request, because I can't reproduce this problem. I'll consider it later if you already have fixes

Are you sure that's problem with `// Code` comment line ? this debugged at runtime ?

**Can I see .il** instructions when it fail ?

to get it manually, use for example:

```bash
ildasm /quoteallnames /unicode /nobar /linenum "/out:lib.il" "lib.dll"
```

**Comment by 3F** [comment]
can you check ca1d73a with problem locale ? (see binaries from nightly builds)
probably it was solved along with latest changes ( bug #10 )

if not - please details

**Comment by 3F** [comment]
v1.4 changelog entry:

```
    * FIXED?: Probably fixed bug - `non-English system language - syntax error` Issue #7
```

reopen if needed
